app/views/admin/participants/validations/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :title, "Participant #{t :name, scope: t_scope}" %>
+<% content_for :before_content, govuk_back_link(text: "Back", href: admin_participant_path(@participant_profile)) %>
+
+<span class="govuk-caption-l"><%= @participant_profile.user.full_name %>
+<h1 class="govuk-heading-l"><%= t :header, scope: t_scope %></h1>
+
+<p><%= t :description, scope: t_scope %></p>
+
+<%= form_for @decision, url: {}, method: :post do |f| %>
+  <%= f.govuk_radio_buttons_fieldset :approved, inline: true, legend: { text: "Change this decision", tag: 'h2', size: 'm' } do %>
+    <%= f.govuk_radio_button :approved, 1, label: { text: 'Approved' }, checked: @decision.approved %>
+    <%= f.govuk_radio_button :approved, 0, label: { text: 'Rejected' }, checked: @decision.approved == false %>
```

Comment:
Let's use double quotes everywhere please :)

Reply:
Uh, shall we add erb-lint?

Reply:
Maybe. Perhaps not in this PR ;)